src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService.cs

```diff
@@ -199,6 +199,7 @@ protected static T Cast<T>(object value)
             {
                 newDocument = await ImportAdder.AddImportsFromSymbolAnnotationAsync(
                     newDocument,
+                    safe: true,
```

Comment:
is there a good reason to have this option?  when would it be desirable to pass `safe: false`?

Reply:
I've kept this option to preserve backwards comparability, and also because it's more performant.
It's easy to remove though, if you think that's a better idea.

Safe can also eg. expand an alias, which then wont get simplified back to the alias, so it's not always desirable.